models/helpers.ts

```diff
@@ -160,6 +161,27 @@ export function getLedgerLink(
     },
   };
 }
+export function getMakeReturnDocAction(fyo: Fyo): Action {
+  return {
+    label: fyo.t`Return`,
+    group: fyo.t`Create`,
+    condition: (doc: Doc) =>
+      !!fyo.singles.AccountingSettings?.enableStockReturns &&
+      doc.isSubmitted &&
+      !doc.isCancelled &&
```

Comment:
This is not necessary as `isSubmitted` is true only if it isn't cancelled

Reply:
corrected